postgrid/__init__.py

```diff
@@ -470,6 +479,95 @@ def construct_event(cls, payload, secret):
             raise ValueError()
 
 
+class RetrieveableAVResource:
+    @classmethod
+    def retrieve(cls, locals_):
+        assert 'cls' in locals_
+
+        locals_except_cls = {
+            key: value for key, value in locals_.items() if key != 'cls'
+        }
+
+        return _av_get(cls.endpoint, **locals_except_cls)
+
+
+class CreateableAVResource:
```

Comment:
Could probably rename this to `PostableAVResource` or something since nothing gets created in AV. Similar for above.

Reply:
fixed.